lib/run.js

```diff
@@ -29,7 +29,11 @@ function start() {
   probot.start();
 }
 
-throng({
-  workers,
-  lifetime: Infinity,
-}, start);
+if (workers === 1) {
```

Comment:
This is what we do in the Slack integration to make debugging locally easier (only 1 process, throng with 1 worker still starts 2 processes).